### DISCUSSION TITLE
Changing font parameters of title on main page (index.html)

### DISCUSSION THREAD

**Discussion by lauralgrima** [daattali/beautiful-jekyll#1084] [Q&A]
I've tried changing a bunch of values in the in the beautifuljekyll.css file and seem to be able to change font parameters for everything but the title on the main page (i.e. "beautiful jekyll" on beautifuljekyll.com). I've also tried adding something like "font: Arial" to the index.html file which doesn't seem to be doing anything. 

Thanks for any guidance!

**Comment by daattali** [comment]
The font for the title comes from https://github.com/daattali/beautiful-jekyll/blob/848d04c90ac87555ccbd00797ce26ee2bd92ecdf/assets/css/beautifuljekyll.css#L29-L30

If you want to change it only on the index page, then you'll need to add a CSS file that only gets loaded on that page (using the `css` YAML parameter) and have a rule such as `h1 { font-family: Arial }`
